ovn-tester/ovn_workload.py

```diff
@@ -790,9 +869,50 @@ def start(self):
         self.nbctl.set_global(
             'northd_probe_interval', self.cluster_cfg.northd_probe_interval
         )
+        self.nbctl.set_global_name(f'az{self.central_node.id + 1}')
```

Comment:
I don't really like the + 1 in various places.  Let's try to make it such that central_node.id is the actual AZ number.  That should help avoid making off-by-one mistakes.

Reply:
@dceara ack, I will fix it